test/metadata.test.js

```diff
@@ -91,6 +101,16 @@ describe('Metadata', () => {
         '[Metadata.tokenLevelMetadata] "animation_url" must be a valid uri with a scheme matching the ipfs|https? pattern',
       );
     });
+    it('should throw error if trait_type is not defined', () => {
+      expect(() =>
+        Metadata.OpenSeaTokenLevelStandard({
+          name: faker.datatype.string(),
+          description: faker.datatype.string(),
+          image: `ipfs://${faker.datatype.uuid()}`,
+          attributes: [{ value: faker.datatype.number() }],
+        }),
+      ).toThrow('[Metadata.tokenLevelMetadata] "attributes[0].trait_type" is required');
```

Comment:
Not part of this PR, but `tokenLevelMetadata` should have been renamed in `OpenSeaTokenLevelStandard`. Can you update the error type ?